src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1006,7 +1021,8 @@ void cbmc_parse_optionst::help()
     "\n"
     "Semantic transformations:\n"
     // NOLINTNEXTLINE(whitespace/line_length)
-    " --nondet-static              add nondeterministic initialization of variables with static lifetime\n"
+    " --nondet-static              add nondeterministic initialization of variables with static lifetime\n" // NOLINT(*)
+    " --nondet-static-except e     same as nondet-static except variables listed in e\n" // NOLINT(*)
```

Comment:
As @smowton, I have a preference for the suffix `-exclude`. Also, comma-separation should not be used, but multiple instances of the option should be supplied instead.